fe/fe-core/src/main/java/com/starrocks/journal/JournalEntity.java

```diff
@@ -202,7 +200,7 @@ public void readFields(DataInput in) throws IOException {
                 break;
             }
             case OperationType.OP_CREATE_WH: {
```

Comment:
How about removing all these warehouse-related edit log?

Reply:
CREATE_WH can not be removed，because we create default warehouse，so  it is necessary to logEdit and replay it after fe restarted.

Reply:
Could the default warehouse be created without any edit log?